AuthSamples/Sample/MainViewController.m

```diff
@@ -539,10 +535,6 @@ - (id)initWithNibName:(nullable NSString *)nibNameOrNil bundle:(nullable NSBundl
                                                  name:FIRAuthStateDidChangeNotification
                                                object:nil];
     self.useStatusBarSpinner = YES;
-#if INTERNAL_GOOGLE3_BUILD
-    // Trigger automatic token refresh.
-    [[FIRDatabase database] reference];
```

Comment:
Removing this line means that we will no longer be triggering automatic token refresh in the sample app.